app/matplotlib.ipynb

```diff
@@ -306,23 +320,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
-   "outputs": [],
    "source": [
+    "from matplotlib import cm\n",
+    "\n",
     "x = np.linspace(-5, 5, 100)\n",
     "y = np.linspace(-5, 5, 100)\n",
     "X, Y = np.meshgrid(x, y)\n",
     "Z = X**2 + Y**2\n",
     "\n",
-    "# write your code here\n"
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "\n",
+    "ax = fig.add_subplot(111, projection=\"3d\")\n",
+    "\n",
+    "surf = ax.plot_surface(X, Y, Z, cmap=cm.coolwarm, linewidth=0, antialiased=False)\n",
```

Comment:
The `cmap` is set to `cm.coolwarm`, but the task suggests using `viridis`. Consider changing it to `cmap=cm.viridis`.